src/main/java/edu/isi/oba/MapperOperation.java

```diff
@@ -131,7 +133,7 @@ private void setOperationGet() {
                 .schema(new StringSchema()));
         parameters.add(new QueryParameter()
                 .name("page")
-                .description("The page number")
+                .description("Number of pages")
```

Comment:
Nop. This is the page that do you want.

I want the page 1.

Reply:
So, change it for "Page number"?

Reply:
Ok